api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -107,6 +110,20 @@ public void validate(Object target, Errors errors) {
 				validateStopDatetime(visit, otherVisit, errors);
 			}
 		}
+        	if(visit.getStartDatetime()!=null && visit.getPatient()!=null && visit.getPatient().getBirthdate()!=null)
+        	{
+            		Date birthDateTime = visit.getPatient().getBirthdate();
+            		Date startDateTime = visit.getStartDatetime();
+            		if(startDateTime.compareTo(birthDateTime) < 0) {
+                		errors.rejectValue("startDatetime","Visit.error.startDateBeforeBirthDate");
+            		}
+            		else {
```

Comment:
Can you put a comment here explaining what you are trying to do in the next two lines below?
Or simply wrap it in a self descriptive method?